development/skinning.md

```diff
@@ -0,0 +1,182 @@
+# Apply skinning to the Alpheios Tools
+
+All branding could be changed using various variables.
+You should define your values of variables and apply them to the `:root` tag.
+For example this way:
+```
+:root {
+    --alpheios-toolbar-bg-color: #008484;
+    --alpheios-border-color: #dedede;
+    --alpheios-desktop-popup-progress-bar-line: #405480;
+
+    --alpheios-icon-color: #BCE5F0;
+    --alpheios-icon-color-hover: #fce45c;
+    --alpheios-icon-color-active: #fce45c;
+    --alpheios-icon-color-disabled: #dedede;
+    --alpheios-settings-font-size-dark-color: #337ab7;
+    --alpheios-settings-font-multiselect-bg: #337ab7;
+}
+
+```
+
+There are several variables that allow to change colors for the Alpheios Tools:
+
+Variable name | Default | Description
+------------ | ------------- | -------------
+`--alpheios-toolbar-bg-color` | #0E2233 | It is a dark color used for popup, panel headers and for the background of floating toolbar
+`--alpheios-border-color` | #757575 | It is used for toolbar border and lookup component border
+ |  | 
+`--alpheios-icon-color` | #FFF | It is used for toolbar, popup and panel icons (desktop and compact)
+`--alpheios-icon-color-hover` | #FFD27D | State color for icon color - hover
+`--alpheios-icon-color-active` | #FFC24F | State color for icon color - active, pressed
+`--alpheios-icon-color-disabled` | #757575 | State color for icon color - disabled
+`--alpheios-icon-color-disabled` | #757575 | State color for icon color - disabled

```

Comment:
This line is a duplicate of a previous one

Reply:
fixed